docs/getting-started-scalardb-analytics-postgresql.md

```diff
@@ -0,0 +1,511 @@
+   ```
+
+   The password is `scalardb-analytics`.
+
+1. Read sample data in the `customer.customers` table.
+
+   ```sql
+   SELECT * FROM customer.customers;
+   ```
+
+   [Command execution result]
+
+   ```sql
+    customer_id |     name      | credit_limit | credit_total
+   -------------+---------------+--------------+--------------
+              1 | Yamada Taro   |        10000 |        10000
+              2 | Yamada Hanako |        10000 |         9500
+              3 | Suzuki Ichiro |        10000 |         8500
+   (3 rows)
+   ```
+
+1. Read sample data in the `order.orders` table.
+
+   ```sql
+   SELECT * FROM "order".orders;
+   ```
+
+   [Command execution result]
+
+   ```sql
+   scalardb=# SELECT * FROM "order".orders;
+    customer_id |   timestamp   |               order_id
+   -------------+---------------+--------------------------------------
+              1 | 1700124015601 | 5ae2a41b-990d-4a16-9700-39355e29adf8
+              1 | 1700124021273 | f3f23d93-3862-48be-8a57-8368b7c8689e
+              2 | 1700124028182 | 696a895a-8998-4c3b-b112-4d5763bfcfd8
+              2 | 1700124036158 | 9215d63a-a9a2-4471-a990-45897f091ca5
+              3 | 1700124043744 | 9be70cd4-4f93-4753-9d89-68e250b2ac51
+              3 | 1700124051162 | 4e8ce2d2-488c-40d6-aa52-d9ecabfc68a8
+              3 | 1700124058096 | 658b6682-2819-41f2-91ee-2802a1f02857
+              3 | 1700124071240 | 4e2f94f4-53ec-4570-af98-7c648d8ed80f
+   (8 rows)
+   ```
+
+1. Read sample data in the `order.statements` table.
+
+   ```sql
+   SELECT * FROM "order".statements;
+   ```
+
+   [Command execution result]
+
+   ```sql
+   scalardb=# SELECT * FROM "order".statements;
+                  order_id               | item_id | count
+   --------------------------------------+---------+-------
+    5ae2a41b-990d-4a16-9700-39355e29adf8 |       2 |     2
+    5ae2a41b-990d-4a16-9700-39355e29adf8 |       1 |     3
+    f3f23d93-3862-48be-8a57-8368b7c8689e |       5 |     1
+    696a895a-8998-4c3b-b112-4d5763bfcfd8 |       4 |     1
+    696a895a-8998-4c3b-b112-4d5763bfcfd8 |       3 |     1
+    9215d63a-a9a2-4471-a990-45897f091ca5 |       2 |     1
+    9be70cd4-4f93-4753-9d89-68e250b2ac51 |       1 |     1
+    4e8ce2d2-488c-40d6-aa52-d9ecabfc68a8 |       2 |     1
+    658b6682-2819-41f2-91ee-2802a1f02857 |       3 |     1
+    4e2f94f4-53ec-4570-af98-7c648d8ed80f |       5 |     1
+   (10 rows)
+   ```
+
+1. Read sample data in the `order.items` table.
+
+   ```sql
+   SELECT * FROM "order".items;
+   ```
+
+   [Command execution result]
+
+   ```sql
+   scalardb=# SELECT * FROM "order".items;
+    item_id |  name  | price
+   ---------+--------+-------
+          5 | Melon  |  3000
+          2 | Orange |  2000
+          4 | Mango  |  5000
+          1 | Apple  |  1000
+          3 | Grape  |  2500
+   (5 rows)
+   ```
+
+1. Run the `JOIN` query.
+
+   For example, you can see the order history of each user as follows.
```

Comment:
```suggestion
1. Run the `JOIN` query. For example, you can see the order history of each user as follows.
```

Reply:
Ah, sorry. I forgot to update this explanation when I updated the SQL query in another comment.
So, I fixed it in 096a22321efd9663f03dd227331e0a48ca413dda .